### DISCUSSION TITLE
SSO authentication has disappeared for Snowflake drivers in 22.3.1

### DISCUSSION THREAD

**Discussion by kjn70** [dbeaver/dbeaver#18661] [General]
### Description

I had Dbeaver Community Edition 22.3.0 installed on my computer as admin (for all users) for Windows.  I updated to 22.3.1 as a fresh install for local user only and once I verify that my connections were present, uninstalled 22.3.0. Opening 22.3.1 and then selecting a connection caused it to throw an authentication issue.  Investigating, I discovered that SSO connection options have disappeared.  

Connection options in 22.3.0 are SSO (Browser) or Database Native.  Connections options for 22.3.1 are only database native with authenticator options of Snowflake or external browser.  For our organization this does not work as we need SSO.  I have had to uninstall 22.3.1 and then reinstall 22.3.0 as a local user which returned the SSO functionality.

### DBeaver Version

Community Edition 22.3.1

### Operating System

Window 10

### Database and driver

snowflake-jdbc-3.13.26.jar

**Comment by Matvey16** [comment]
That was done intententionally as part of our extended support of Snowflake databases. Right now SSO authentication through UI as for most of other databases is exclusive to our PRO versions which you can try for free on [dbeaver.com](https://dbeaver.com/trial/)
Configuring SSO authentication through JDBC driver remains anaffected, so it can be manually configured, only the UI feature was removed
